internal/server/add_payment_test.go

```diff
@@ -285,4 +285,4 @@ func TestPostAddPaymentAmountIncorrectFormat(t *testing.T) {
 			mock.AssertExpectationsForObjects(t, client, template)
 		})
 	}
-}
+}
```

Comment:
Sorry this is a picky one, but if you could add the blank EOF line back in it'd just keep the linters happy :) Same in `add_payment.go`